contrib/kiss-outdated

```diff
@@ -26,14 +26,27 @@ EOF
     )
 
     case $remote in
+        age)
+            remote=age-encryption
+        ;;
+
         baseinit|baselayout|kiss)
             remote=-
         ;;
 
+
+        bat)
+            remote=bat-cat
+        ;;
+
         clang)
             remote=llvm
         ;;
 
+        ctags)
+            remote=ctags-unclassified
```

Comment:
`exuberant-ctags` seems to be the remote name: https://repology.org/project/exuberant-ctags/versions